products/content/tier-2-content-IA-and-design/cope-cms-faq-library/engineering/README.md

```diff
@@ -0,0 +1,64 @@
+# Exposing Q&A on New Content Types
+
+## Pre-Existing Setup
```

Comment:
This doc should ideally contain both Drupal and FE steps, when @chri5tia 's  BE ticket is done. Can we add a header to clarify that this is the FE flow?  Otherwise, LGTM if @jtmst signs off on technical accuracy.

Reply:
Updated the headers. Should I wait to merge this until @chri5tia's BE ticket is complete? Or will she just be adding to this when she's done?

Reply:
Adding a note here that Josh re-approved after the code changes were updated.
<img width="767" alt="Screenshot 2023-06-30 at 12 27 19 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/19175324/64c34911-e9e1-4ba6-b4f5-8028c6d0ec11">